types/sinon/ts3.1/sinon-tests.ts

```diff
@@ -87,6 +87,10 @@ function testSandbox() {
     const privateFooFoo: sinon.SinonStub = privateFooStubbedInstance.foo;
     const clsBar: number = stubInstance.bar;
     const privateFooBar: number = privateFooStubbedInstance.bar;
+    sb.createStubInstance(cls, {
+        foo: (arg1: string, arg2: number) => 2,
```

Comment:
This test doesn't doesn't look correct. According to the docs, the value should either be a stub object or a return value to be used, i.e. here `SinonStub | number`, but it is a function